requirements.txt

```diff
@@ -26,4 +26,9 @@ ibm-watson==4.4.0
 websockets==8.1
 webexteamssdk==1.3
 voluptuous==0.11.7
+bitstring==3.1.7 
 regex==2020.4.4
+bitstring==3.1.7 
+=======
```

Comment:
I believe I didn't explain well what you need to do here, but you need to remove this line 
```suggestion

```